setup.py

```diff
@@ -29,7 +29,6 @@ def load_requirements():
         'Intended Audience :: Developers',
         'Environment :: Web Environment',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2.7',
```

Comment:
Should be accompanied with something like (not sure about the actual constraint):

```python
    python_requires=">=3.6",
```

Reply:
I don't think you need to specify this. At least I don't see anything like this on my other project.
It should be enough that there is `'Programming Language :: Python :: 3'`

Reply:
It is needed to avoid PIP on Python 2.7 to install such package.

Reply:
Added `python_requires`